### PR TITLE
Changes necessary to support AMI building

### PR DESCRIPTION
**Pull request by michaelgugino**
Currently, AMI building does not work.

These changes implement the necessary steps to build
an AMI for usage as a 'golden node image'.

### CODE REVIEW

File: playbooks/aws/openshift-cluster/build_ami.yml [hunk 1]
```diff
@@ -66,8 +66,14 @@
...
 
-- name: install node config
-  include: ../../common/openshift-node/config.yml
+- name: run node config setup
+  include: ../../common/openshift-node/setup.yml
+
+- name: run node config
+  include: ../../common/openshift-node/configure_nodes.yml
+
+- name: Re-enable exlcuders
```
Comment (kwoodson) [inline playbooks/aws/openshift-cluster/build_ami.yml]: excluders*

File: roles/openshift_node/defaults/main.yml [hunk 2]
```diff
@@ -31,12 +31,9 @@ openshift_node_ami_prep_packages:
 - python-dbus
 - PyYAML
 - yum-utils
-- python2-boto
-- python2-boto3
 - cloud-utils-growpart
 # gluster
 - glusterfs-fuse
-- heketi-client
```
Comment (sdodson) [inline roles/openshift_node/defaults/main.yml]: why is this removed?

File: roles/openshift_node/defaults/main.yml [hunk 3]
```diff
@@ -31,12 +31,9 @@ openshift_node_ami_prep_packages:
 - python-dbus
 - PyYAML
 - yum-utils
-- python2-boto
-- python2-boto3
```
Comment (sdodson) [inline roles/openshift_node/defaults/main.yml]: these make sense to remove

File: roles/openshift_node/defaults/main.yml (hunk 2 above)
Comment (michaelgugino) [reply roles/openshift_node/defaults/main.yml]: This package doesn't exist in origin (centos) repos.

Either this needs to be removed, or we need to refactor how this package is installed, eg, openshift-enterprise vs origin.

I think having this list of packages in defaults/main.yml of openshift_node is a big work-around.  We need to refactor several roles to ensure all the right configs and packages are in place for AMI building, not just bulk-install packages here.

File: roles/openshift_node/defaults/main.yml (hunk 3 above)
Comment (michaelgugino) [reply roles/openshift_node/defaults/main.yml]: These also aren't available in origin.

Centos decided to package these as 'python-boto'.  There is no package python-boto3 in Centos, at least not in the repos we're consuming.

File: roles/openshift_node/defaults/main.yml (hunk 2 above)
Comment (sdodson) [reply roles/openshift_node/defaults/main.yml]: ok
